src/fondant/components/retrieve_images_from_faiss_index/src/main.py

```diff
@@ -0,0 +1,71 @@
+import typing as t
+
+import dask.dataframe as dd
+import faiss
+import fsspec
+import pandas as pd
+from fondant.component import DaskTransformComponent
+from transformers import AutoTokenizer, CLIPTextModelWithProjection
+
+ID_MAPPING = "id_mapping"
+FAISS_INDEX = "faiss3"
```

Comment:
These shouldn't be hardcoded since they are variables in `autofaiss`. Should we just accept two paths?